src/bootstrap.js

```diff
@@ -14,10 +18,25 @@ function install() {} // eslint-disable-line no-unused-vars
 function uninstall() {} // eslint-disable-line no-unused-vars
 
 function startup() { // eslint-disable-line no-unused-vars
+  Services.prefs.setBoolPref(UA_ENABLE_PREF_NAME, true);
+
+  Services.prefs.addObserver(UA_ENABLE_PREF_NAME, () => {
+    let isEnabled = Services.prefs.getBoolPref(UA_ENABLE_PREF_NAME);
+    if (isEnabled && !overrider) {
+      overrider = new UAOverrider(UAOverrides);
+      overrider.init();
+    } else if (!isEnabled && overrider) {
+      overrider.uninit();
+      overrider = false;
+    }
+  }, false);
+
   overrider = new UAOverrider(UAOverrides);
   overrider.init();
 }
 
 function shutdown() { // eslint-disable-line no-unused-vars
-  overrider.uninit();
+  if (overrider) {
+    overrider.uninit();
+  }
```

Comment:
Another concern when shutdown, you can't use Services.prefs.removeObserver when it's an anonymous function. I suggest move observer to an independent function for removeObserver.